src/components/Reviews.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+class Reviews extends Component {
```

Comment:
use a functional component

Reply:
You've not fixed this. Always do all the changes requested in a PR before requesting for another review.

Reply:
Done sir